runtime/onert/core/src/loader/BaseLoader.h

```diff
@@ -94,7 +94,7 @@ template <typename LoaderDomain> class BaseLoader
 
   // Helper functions
   ir::Activation convertActivation(ActivationFunctionType type);
-  ir::DataType tensorTypeToDataType(TensorType type);
+  virtual ir::DataType tensorTypeToDataType(const TensorType type);
```

Comment:
@hseok-oh 

In #13693, you implemented
```c
template <typename LoaderDomain>
ir::DataType BaseLoader<LoaderDomain>::BaseLoader::getTensorDataType(const Tensor *tensor)
{
  return tensorTypeToDataType(tensor->type());
}
```
Maybe for tflite loader.

However, it is missing in this PR.

Is there any reason?

Reply:
That was introduced to use quantization parameter in tensor object not only data type, but that function is not required any more because we decided to use data type only.